fullstaq-labs/fullstaq-ruby-infra
# Azure Blob Storage container for storing Server Edition CI/CD caches

## Description

Supports https://github.com/fullstaq-labs/fullstaq-ruby-server-edition/issues/86

## Testing instructions

 1. Modify fullstaq-ruby-server-edition CI to use sccache to cache to the storage container.
 2. Trigger a fullstaq-ruby-server-edition CI run.
 3. Verify that the storage container is non-empty.

## Checklist for the submitter



 - [x] Infrastructure overview diagram updated
   
 - [x] The following content/files are kept in sync with each other:
   + The Terraform version specified in `**/providers.tf`, `.github/workflows/code-reviews.yml` and `docs/required-devtools.md`
   + The list of responsibilities specified in `.github/ISSUE_TEMPLATE/apply_join_team.md` and `docs/responsibilities-expectations.md`

## Checklist for reviewers

- [x] If `terraform-hisec/` was updated: someone with the "Infra Owner" role has checked the output of `terraform plan`

## Definition of done (for team members)

This checklist is for the person who merges this pull request. _Tick check boxes, or strike out irrelevant ones._

 - [x] If `terraform-hisec/` was updated: someone with the "Infra Owner" role has run `terraform apply`
 - [x] Tested in production
